resources/js/Pages/AddUser.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <app-layout>
+        <div class="py-12">
+            <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+                <div class="bg-white overflow-hidden shadow-xl sm:rounded-lg">
+                    <create-user-form/>
+                </div>
+            </div>
+        </div>
+    </app-layout>
+</template>
+
+<script>
+import CreateUserForm from "./Admin/Users/CreateUserForm";
+import AppLayout from './../Layouts/AppLayout';
```

Comment:
```suggestion
import AppLayout from '@src/Layouts/AppLayout';
```

For consistency, please use the @src alias when accessing components above the current directory.

Reply:
Done

Reply:
It seems this change is still missing :) @EvanDime